app/helpers/vacancies_helper.rb

```diff
@@ -45,7 +45,7 @@ def uncapitalize_words(location_name)
 
   def new_sections(vacancy)
     sections = []
-    sections << 'job_role' unless vacancy.job_roles&.any?
```

Comment:
I wonder, do we have an agreed principle for when we stop showing the new-section notification? Is it after some time period or is it when the user hasn't seen the section before?

Am I misremembering, or is job_role actually a newer section than supporting_documents is?

Reply:
Indefinitely. There is a ticket for refining how this works I believe